Resources/doc/index.md

```diff
@@ -85,7 +85,7 @@ security:
         - { path: ^/api,       roles: IS_AUTHENTICATED_FULLY }
 ```
 
-Configure your `routing.yml` :
+Configure your `config/routes.yml` :
```

Comment:
shouldn't it be `config/routes.yaml`?

Reply:
that is correct :) I will change it. thanks @chalasr